statsmodels/tsa/arima_model.py

```diff
@@ -732,7 +747,10 @@ def predict(self, params, start=None, end=None, exog=None, dynamic=False):
                 exog = exog[:, None]
 
         if out_of_sample != 0 and self.k_exog > 0:
-            # we need the last k_ar exog for the lag-polynomial
+            exog = np.asarray(exog)
+            if self.k_exog == 1 and exog.ndim == 1:
+                exog = exog[:, None]
+                # we need the last k_ar exog for the lag-polynomial
```

Comment:
this part here is redundant/duplicate. It is now included in the lines just above under `if exog is not None`